tyk-control-plane/values.yaml

```diff
@@ -1131,6 +1124,30 @@ tyk-mdcb:
     # affinity for MDCB pod assignment
     affinity: {}
 
+    tls:
+      # enables ssl for mdcb
+      useSSL: false
+
+      # the path to where the keys will be mounted in the pod
+      certificatesMountPath: "/etc/certs"
+
+      # location to pem encoded private key
+      certificateKeyFile: "/etc/certs/tls.key"
+
+      # location to pem encoded certificate
+      certificateCertFile: "/etc/certs/tls.crt"
+
+      # Sets the AES256 secret which is used to encode certificate private keys when
+      # they uploaded via certificate storage.
+      # It is used to set TYK_MDCB_SECURITY_PRIVATECERTIFICATEENCODINGSECRET
+      privateCertificateEncodingSecret: ""
```

Comment:
Since this field value should match with gateway value should we move it to global?